interface/super/edit_globals.php

```diff
@@ -64,7 +64,7 @@ function checkCreateCDB()
         $couch = new CouchDB();
         if (!$couch->check_connection()) {
             echo "<script type='text/javascript'>alert('".addslashes(xl("CouchDB Connection Failed."))."');</script>";
-            return;
+            return true;
```

Comment:
seems like this would make more sense to be `return false` since this happens when there is a broken connection. note the function call doesn't use this (whether it returns true or false), but would make more sense in case somebody checks for that in the future.